doc/source/cookiecutter_guide.rst

```diff
@@ -2,10 +2,15 @@
 
 .. index:: cookiecutter_guide
     
+
+.. _cookiecutter-title:
+
 ===============================
-How to migrate existing package
+How to cookiecut Python package
```

Comment:
Having a general name since this page contain all information that can be used by both migration and new project